quickget

```diff
@@ -921,7 +921,7 @@ EOF
             fi
             ;;
           vanillaos)
-          ## Minimum 50G for abroot
+          ## Minimum 64G for abroot
```

Comment:
The minimum for abroot is 50G
`quickget` adds some extra headroom but the comment documents the true minimum

Please revert this change, optionally with a mention of the 64G allocation with headroom if you wish

Reply:
Thanks for explaining the two limits. Mentioned it in the comment in the next commit.